modules/permutiveRtdProvider.js

```diff
@@ -341,6 +373,21 @@ function iabSegmentId(permutiveSegmentId, iabIds) {
   return iabIds[permutiveSegmentId] || unknownIabSegmentId
 }
 
+/**
+ * Passes Permutive's standard cohorts to Appnexus/Xandr via auction level keywords
+ * @param {Object} sspData -- {ssp?: string[], cohorts?: string[]}
+ */
+function passCohortsToAppnexusAuctionKeywords(sspData) {
```

Comment:
This is a rules violation, you must write to the standard object and appnexus adapter should be responsible for the mutation